src/main/java/com/exasol/adapter/document/mapping/AbstractPropertyToNumberColumnMapping.java

```diff
@@ -5,12 +5,9 @@
 import com.exasol.adapter.document.edml.ConvertableMappingErrorBehaviour;
 import com.exasol.adapter.document.edml.MappingErrorBehaviour;
 
-import lombok.experimental.SuperBuilder;
-
 /**
  * Abstract base for {@link AbstractPropertyToColumnMapping}s that map to numeric Exasol columns.
  */
-@SuperBuilder(toBuilder = true)
```

Comment:
Doesn't this class now need a method `toBuilder()`?

Reply:
No, it's not necessary here